### ISSUE TITLE
💡Feature Request: Consistent User Types

### ISSUE THREAD

**Issue by cnrmck** [alex-grover/neynar-next#36]
`Cast.author` has a bespoke definition, so the response from the getUser endpoint returns a different `User` type than the author field in the getCast endpoint, it would be easier to interface with if `Cast.author` was the same User definition as the standard `User` type

I.e. switch from:
```javascript
export type Cast = {
  hash: Hash
  thread_hash: Hash | null
  parent_hash: Hash | null
  parent_url: string | null
  parent_author: {
    fid: number | null
  }
  author: {
    fid: number
    username: string
    display_name: string
    pfp_url: string
    profile: {
      bio: {
        text: string
      }
    }
    follower_count: number
    following_count: number
    verifications: Address[]
    active_status: 'active' | 'inactive'
  }
  text: string
  timestamp: string
  embeds: Embed[]
  reactions: {
    likes: Like[]
    recasts: Recast[]
  }
  replies: {
    count: number
  }
}
```
To

```javascript
export type Cast = {
  hash: Hash
  thread_hash: Hash | null
  parent_hash: Hash | null
  parent_url: string | null
  parent_author: {
    fid: number | null
  }
  author: User
  text: string
  timestamp: string
  embeds: Embed[]
  reactions: {
    likes: Like[]
    recasts: Recast[]
  }
  replies: {
    count: number
  }
}
```

**Comment by alex-grover** [comment]
Released in v0.3.0!

**Comment by cnrmck** [comment]
Yep, it works!
